### PR TITLE
change forEach to reduce in home.js route

### PR DESCRIPTION
**Pull request by philawsophizing**
sorry took me a minute had to re-learn it quickly! I guess reduce is better/more explicit..? but seems like more code/effort in a way

### CODE REVIEW

File: src/routes/home.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ const NUM_OF_RESULTS = 50;
 
 const handler = (request, reply) => {
   dbQueries.retrieveTransactions(connPool, (err, response) => {
-    let total = 0;
-    response.rows.forEach(function(row) {
-      total += row.transaction_value;
-    });
+    // Sum the transactions in the kitty
+    let total = response.rows.reduce(function(acc, row) {
```
Comment (Karyum) [inline src/routes/home.js]: what is acc? 
and maybe use es6 for consistency?

**Review by Karyum — CHANGES_REQUESTED** [review]

**Comment by philawsophizing** [comment]
@Karyum check out changes

**Review by Karyum — APPROVED** [review]
great!

**Comment by jwld** [comment]
#32

**Pull request merged**